flashlight/app/asr/CMakeLists.txt

```diff
@@ -62,9 +62,13 @@ add_executable(fl_asr_train ${CMAKE_CURRENT_LIST_DIR}/Train.cpp)
 add_executable(fl_asr_test ${CMAKE_CURRENT_LIST_DIR}/Test.cpp)
 add_executable(fl_asr_decode ${CMAKE_CURRENT_LIST_DIR}/Decode.cpp)
 
-target_link_libraries(fl_asr_train flashlight-app-asr ${CMAKE_DL_LIBS})
-target_link_libraries(fl_asr_test flashlight-app-asr ${CMAKE_DL_LIBS})
-target_link_libraries(fl_asr_decode flashlight-app-asr ${CMAKE_DL_LIBS})
+if (NOT FL_LIBRARIES_USE_MKL)
```

Comment:
Instead of linking this to binaries explicitly, the solution here is to make the `target_link_libraries` call for OpenMP `PUBLIC` so that targets that link to flashlight will have it transitively included automatically.

Reply:
fixed